portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,19 +34,28 @@
     
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-     //retrieves comment data from datastore
+     // Get the user input on how many comments to display
+     
+    Integer maxComments = getMaxComments(request);
+     //retrieves comment data from datastore 
```

Comment:
Nit: Capitalize the first letter in each comment, end with a period. Makes it easier to tell if the comment was truncated on accident.